src/elements/helpers/Preview.tsx

```diff
@@ -40,23 +85,45 @@ export const Preview = ({
     }
   };
 
+  const listenToMessages = () => {
+    window.addEventListener("message", (message) => {
+      try {
+        const data = JSON.parse(message.data) as unknown;
+
+        if (isMessageData(data) && data.type === "set-height") {
+          const height = data.value;
+          setHeight(`${(height + 4).toString()}px`);
```

Comment:
`nit` It might be nice to move this `4` into a const to be explicit about what it represents.